webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+module.exports = {
+  entry: {main: "./client/src/js/index.js"},
```

Comment:
It'd be better to add __dirname in front of filepath

Reply:
One of the tutorial use relative path